src/components/Table/TableFormatters/DetailFormatter.vue

```diff
@@ -114,20 +114,22 @@ export default {
       return resource.replace(' details', '').replace('详情', '')
     },
     getDrawerTitle() {
-      let drawerTitle = ''
-      if (this.formatterArgs?.getTitle && typeof this.formatterArgs.getTitle === 'function') {
-        drawerTitle = this.formatterArgs.getTitle({
+      if (this.formatterArgs?.getDrawerTitle && typeof this.formatterArgs.getDrawerTitle === 'function') {
+        this.formatterArgs.getDrawerTitle({
           col: this.col,
           row: this.row,
-          cellValue: this.cellValue,
-          index: this.index
+          cellValue: this.cellValue
         })
       }
-      let title = this.cellValue?.name || drawerTitle
+
+      let title = this.cellValue || this.row.name
+
       const resource = this.getResource()
+
       if (resource) {
         title = `${resource}: ${title}`
       }
+
       return title
     },
     resolveRoute() {
```

Comment:
The main issue is that `export default {};` should be removed because it's not needed anymore. This line makes sense before using curly braces `{ }`.

Additionally, there might also be an issue with the usage of function arguments like `cellValue`, and their types. It'd be good to ensure all necessary data types are used instead.

In terms of optimization, it looks fine for now. No further changes required right now based on provided text. However, considering the code's complexity, more extensive checks could be useful depending on your requirements.